app/assets/src/components/views/SampleUploadFlow/UploadMetadataStep.jsx

```diff
@@ -116,7 +118,8 @@ UploadMetadataStep.propTypes = {
   project: PropTypes.shape({
     id: PropTypes.number,
     name: PropTypes.string
-  })
+  }),
+  handleRequiredMetadataFields: PropTypes.func
```

Comment:
Doesn't look like `handleRequiredMetadataFields` is used.

Reply:
oops I think I merged it wrong